practice-app/api/views.py

```diff
@@ -18,10 +21,27 @@ def post(self, request):
             'message': f'Hello, {name}.'
         })
 
+class UptodateParityView(APIView):
+	def get(self, request):
+		return Response({
+			'message': 'Hello, buddy.'
+		})
+
+	def post(self, request):
+		base_symbol = request.data['base_symbol']
+		target_symbol = request.data['target_symbol']
+		base_equipment = Equipment.objects.get(symbol=base_symbol)
+		target_equipment = Equipment.objects.get(symbol=target_symbol)
+		filtered = Parity.objects.filter(base_equipment = base_equipment, target_equipment = target_equipment)
```

Comment:
are you sure just filtering and slicing returns the latest parity? It may not always be the case. For the sake of safety, you might want to add orderby in between objects and filter. Like,
```suggestion
		filtered = Parity.objects.order_by('-date').filter(base_equipment=base_equipment,
	                                                                         target_equipment=target_equipment)
```

Reply:
I suppose since the fake data generator creates parities starting from today back to the past, the items were already by decreasing dates when you were testing this. But this won't be the case when we start using the cronjob. You **must** sort the items by date get the last one as @mertyg said. Or it will fail in the future.